worlds/generic/docs/advanced_settings_en.md

```diff
@@ -191,13 +191,23 @@ triggers:
         bigkey_shuffle: any_world
         map_shuffle: any_world
         compass_shuffle: any_world
+Timespinner:
+  item_links: # Share part of your item pool with other players.
+    - name: TSAll
+      item_pool: 
+        - Everything
+      local_items:
+        - Twin Pyramid Key
+        - Timespinner Wheel
+      replacement_item: null 
+    
 ```
 
 #### This is a fully functional yaml file that will do all the following things:
 
 * `description` gives us a general overview so if we pull up this file later we can understand the intent.
 * `name` is `Example Player` and this will be used in the server console when sending and receiving items.
-* `game` is set to `A Link to the Past` meaning that is what game we will play with this file.
+* `game` has an equal chance of being either `A Link to the Past` or `Timespinner` with a 10/20 chance for each.
```

Comment:
The sentence says they have an equal chance, then that they each have 10/20 chance. Instead of restating chances, please describe in one clause that they are equal 10/20 chances.